documentation/asciidoc/topics/proc_creating_service_monitor.adoc

```diff
@@ -1,52 +1,26 @@
 [id='creating-service-monitor_{context}']
 = Creating a Prometheus Service Monitor
-Create a Prometheus `ServiceMonitor` that scrapes metrics from a {brandname} cluster.
+{ispn_operator} can create a Prometheus `ServiceMonitor` that scrapes metrics from a {brandname} cluster.
```

Comment:
@dmvolod @oraNod Creates automatically if ServiceMonitor resource is available?

Reply:
Yes, @Crumby correct.

Reply:
@Crumby yes, my understanding wasn't 100% but @dmvolod has set me straight. the operator adds the annotation when it detects that the prometheus operator is installed.